Implement VisitorCountsPage, Container, Chart

JamieShin0201 opened the pull request:
![image](https://user-images.githubusercontent.com/65442049/100324076-06fd4580-300a-11eb-91ff-857ab7b67c67.png)

방문자수를 볼수 있도록 구현

JamieShin0201 committed:
Implement VisitorCountsPage, Container, Chart

Install victory
Show visitorCounts chart

hannut91 commented on src/components/VisitorCountsChart.jsx:
```diff
@@ -0,0 +1,56 @@
+  function makeDataforChart(value) {
+    let data = [];
+
+    value.forEach(({
+      date: {
+        year, month, day, hour, min,
+      }, count,
+    }) => {
+      data = [...data, { x: new Date(year, month, day, hour, min), y: count }];
+    });
```
부수 효과가 있군요!

hannut91 commented on src/components/VisitorCountsChart.test.jsx:
```diff
@@ -0,0 +1,21 @@
+
+import VisitorCountsChart from './VisitorCountsChart';
+
+import VISITORCOUNTS from '../../fixtures/visitorCounts';
```
`VISITOR_COUNTS`라고 쓰면 더 잘보일 것 같아요
그리고 VISITORS_COUNT가 더 자연스러울 것 같아요

JamieShin0201 replied:
자료형이 배열인 변수명은 뒤에 s를 붙이는게 좋을거 같아서 VISTOR_COUNTS로 생각했는데 많이 어색할까요?

hannut91 replied:
아하 제가 착각했네요 ㅎㅎ종명님이 말씀하시는게 맞네요 ㅎㅎ 방문자수의 목록이었쬬?

hannut91 commented on src/containers/VisitorCountsContainer.jsx:
```diff
@@ -0,0 +1,32 @@
+
+  const visitorCounts = useSelector(get('visitorCounts'));
+
+  if (!visitorCounts || visitorCounts.length === 0) {
+    return (
+      <p>방문자 수가 없습니다.</p>
+    );
+  }
```
lodash의 `isEmpty`를 써보는건 어떨까요?

See also
  - https://lodash.com/docs/4.17.15#isEmpty

hannut91 commented on src/containers/VisitorCountsContainer.test.jsx:
```diff
@@ -0,0 +1,50 @@
+
+  context('without visitorCounts', () => {
+    given('visitorCounts', () => ([]));
+    it('renders no visitorCounts message', () => {
```
여기 위에 빈 줄 하나를 띄어서 구분해주면 좋을 것 같아요

JamieShin0201 committed:
Fix VisitorCountsPage, Container, Chart

install lodash and setting
fix variable name VISTORCOUNTS -> VISITOR_COUNTS
remove side effect in VisitorChart
